test/app/lib/storage/vault.spec.ts

```diff
@@ -0,0 +1,51 @@
+import {Vault} from "../../../../app/lib/storage/index"
```

Comment:
In #32 I added the [modulePath](https://github.com/witnet/sheikah/blob/master/package.json#L31) option to jest config to be able to write this line as:
```typescript
import {Vault} from "app/lib/storage/index"
```

Reply:
crap, that's a pity, I'll see if there's a way of specifying jest config in a way WebStorm likes it